### ISSUE TITLE
default values for booleans are incorrect

### ISSUE THREAD

**Issue by morgo** [pingcap/docs#4531]
## Error Report

**This repository is ONLY used to solve issues related to DOCS.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. What is the URL/path of the document related to this issue?

https://docs.pingcap.com/tidb/dev/system-variables

2. How would you like to improve it?

Most of the default values for booleans are incorrect, following the merge of https://github.com/pingcap/tidb/pull/20346
For consistency, they should be documented as ON/OFF, not 0/1.

**Comment by TomShawn** [comment]
Would you help create a PR to modify those incorrect boolean values? I am not sure whether all of the 1/0 values should be documented as ON/OFF.

**Comment by morgo** [comment]
If you run `SHOW GLOBAL VARIABLES` the output will be ON/OFF:
```sql
mysql> SHOW GLOBAL VARIABLES;
+--------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Variable_name                                          | Value                                                                                                                                                                                                                                                                                                                                            |
+--------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| allow_auto_random_explicit_insert                      | OFF                                                                                                                                                                                                                                                                                                                                              |
| auto_increment_increment                               | 1                                                                                                                                                                                                                                                                                                                                                |
| auto_increment_offset                                  | 1                                                                                                                                                                                                                                                                                                                                                |
| autocommit                                             | ON                                                                                                                                                                                                                                                                                                                                               |
| automatic_sp_privileges                                | 1                                                                                                                                                                                                                                                                                                                                                |
| avoid_temporal_upgrade                                 | OFF                                                                                                                                                                                                                                                                                                                                              |
| back_log                                               | 80                                                                                                                                                                                                                                                                                                                                               |
| basedir                                                | /usr/local/mysql                                                                                                                                                                                                                                                                                                                                 |
| big_tables                                             | OFF                                                                                                                                                                                                                                                                                                                                              |
| bind_address                                           | *                                                                                                                                                                                                                                                                                                                                                |
| binlog_cache_size                                      | 32768                                                                                                                                                                                                                                                                                                                                            |
| binlog_checksum                                        | CRC32                                                                                                                                                                                                                                                                                                                                            |
..
```

The order should match the output from this command, which is this issue: https://github.com/pingcap/docs/issues/4532

**Comment by TomShawn** [comment]
Addressed in #4587.
